geth/wrapper.py

```diff
@@ -43,9 +43,8 @@ def get_max_socket_path_length():
 
 
 def construct_test_chain_kwargs(**overrides):
-    overrides.setdefault("unlock", "0")
```

Comment:
Are all these removals args that are no longer present with dropping `<v1.13` support?

Reply:
This was for a private chain setup. It's called internally to set up `DevGethProcess`. I'm slimming this down for transitioning from private chain setup to dev mode setup. `mine` is gone, `personal` is gone, unlocking is not longer a thing.